spec/supabase_js_v2.yml

```
@@ -5334,6 +5334,44 @@ pages:
             .download('folder/avatar1.png')
           ```
 
+  storage.from.publicDownload():
+    title: 'from.publicDownload()'
+    $ref: '@supabase/storage-js.packages/StorageFileApi.default.publicDownload'
+    notes: |
+      - RLS policy permissions required1:
+        - `buckets` table permissions: none
+        - `objects` table permissions: `select`
+      - Refer to the [Storage guide](/docs/guides/storage#access-control) on how access control works
```

Ah, I broke this section out into a separate doc so there are probably other links I need to update too.
```suggestion
      - Refer to the [Storage guide](/docs/guides/storage/access-control) on how access control works
```